ext/fg/js/popup-nested.js

```diff
@@ -53,3 +36,40 @@ async function popupNestedInitialize(id, depth, parentFrameId, url) {
         document.body.appendChild(script);
     }
 }
+
+let popupNestedInitialized = false;
+
+async function popupNestedInitialize(id, depth, parentFrameId, url) {
+    if (popupNestedInitialized) {
```

Comment:
I don't think `popupNestedInitialized` is necessary anymore. Please remove.

Reply:
It's called from `float.js`, but I can't think of cases when it would actually be called multiple times except maybe reloading the extension, and this check wouldn't help with that. Maybe there should be a check similar to the one in `search-frontend.js`

```javascript
        const node = document.querySelector(`script[src='${src}']`);
        if (node !== null) { continue; }
```

Reply:
Yeah, that prepare should only be called once due to the `_prepareInvoked` guard, so that should handle it.

Reply:
https://github.com/FooSoft/yomichan/pull/439/commits/98d7e095a5c3533a9a2d787f3ee15b1640f77da0